apps/web/src/middleware.ts

```diff
@@ -1,7 +1,7 @@
 import { authMiddleware } from "@clerk/nextjs";
 
 export default authMiddleware({
-  publicRoutes: ["/", "/api/og"],
+  publicRoutes: ["/", "/api/og", "/api/webhook/ping", "/api/webhook/clerk"],
```

Comment:
We will need to check for placeholders here. I’m doing the same currently

Reply:
I think we need to do with the matcher instead for the middle ware ?